test/Microsoft.Health.Dicom.Web.Tests.E2E/Rest/DataPartitionEnabledTests.cs

```diff
@@ -130,6 +132,48 @@ public async Task GivenDatasetInstancesWithDifferentPartitions_WhenDeleted_OneDe
             Assert.True(response5.IsSuccessStatusCode);
         }
 
+        [Fact]
+        public async Task GivenAnInstance_WhenRetrievingChangeFeedWithPartition_ThenPartitionNameIsReturned()
+        {
+            if (_isUsingRemoteTestServer)
+            {
+                // Data partition feature flag only enabled locally. For Remote servers, feature flag is by default disabled
+                return;
+            }
+
+            var newPartition = TestUidGenerator.Generate();
+            string studyInstanceUID = TestUidGenerator.Generate();
+            string seriesInstanceUID = TestUidGenerator.Generate();
+            string sopInstanceUID = TestUidGenerator.Generate();
+
+
+            DicomFile dicomFile = Samples.CreateRandomDicomFile(studyInstanceUID, seriesInstanceUID, sopInstanceUID);
+            using DicomWebResponse<DicomDataset> response1 = await _client.StoreAsync(new[] { dicomFile }, partitionName: newPartition);
```

Comment:
Do we need to assert this was successful?

Reply:
Ideally yes :). Fixed